src/main/java/org/babi/backend/place/api/PlaceStateSavePlaceApiPreProcessor.java

```diff
@@ -0,0 +1,16 @@
+package org.babi.backend.place.api;
+
+import org.babi.backend.place.domain.Place;
+import org.babi.backend.place.domain.PlaceState;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PlaceStateSavePlaceApiPreProcessor implements SavePlaceApiPreProcessor {
+    @Override
+    public void preProcess(Place place, Authentication authentication) {
+        PlaceState placeState = authentication.getAuthorities().stream()
+                .anyMatch(grantedAuthority -> grantedAuthority.getAuthority().equals("ADMIN")) ? PlaceState.APPROVED : PlaceState.REVIEW;
```

Comment:
I would always compare already given value with passed value in order to avoinf NP exception 
```suggestion
                .anyMatch(grantedAuthority -> "ADMIN".equals(grantedAuthority.getAuthority()) ? PlaceState.APPROVED : PlaceState.REVIEW;
```